### DISCUSSION TITLE
Line Green Medal

### DISCUSSION THREAD

**Discussion by JohnnyXiao1024** [1101-GS4719-C/Python-Community-GS4719-C#1196] [General]
有鑒於我本週頗閒，再加上快放假的關係，一整個大發懶
雖然有一堆該讀的文本，但又覺得沒什麼動力....

所以想說換個口味
去準備一下上個月意外看到的LINE線上認證
之前其實有玩過LINE OA和LAP 但因為都自己亂玩 學得挺混亂的
想說給自己一個比較完整的練習跟受訓機會

簡單排個五天計畫後 看一下他們的教學錄影帶和各種操作手冊
不得不承認 無論是週一參加的AWS Educate workshop
還是現在的LINE Green Medal 自學歷程

這類由認證考試延伸出的自學資源真的非常豐富
也蠻慶幸我活在一個學習資源豐富又容易取得的時代
（讓我可以常常換個學習主題wwwww）

btw 我的LAP認證差一題通過.....
還要等7天後補考QQ

有興趣的 傳送門在這：[https://greenmedal.linebiz.com/](https://greenmedal.linebiz.com/?fbclid=IwAR388WKiV6TxIqJHJSPg55hiYzVVtmRVG8RJbSqe_BcCTMmrzDqS-Rv0v1Q)

<img width="1056" alt="截圖 2023-03-31 下午5 00 40" src="https://user-images.githubusercontent.com/125842539/229085053-7ae8a154-a10e-4cae-97ec-abb803f1e264.png">
<img width="849" alt="截圖 2023-03-31 下午5 01 00" src="https://user-images.githubusercontent.com/125842539/229085067-dce8419d-0ca9-432e-807e-bca3472dee7e.png">

**Comment by zilinng** [comment]
感謝作者提供那麼有用的資訊！之前在一門通識課上曾經玩過一點點LineOA,當時雖然覺得很好玩但課程結束後完全沒有想到還能依靠網路上的自學資源繼續精進！我想我會好好利用這個連假體驗一下line green medal的課程。
